src/com/lilithsthrone/game/Properties.java

```diff
@@ -685,6 +708,21 @@ public void loadPropertiesFromXML(){
 					}
 				}
 				
+				// Sexual orientation preferences:
+				nodes = doc.getElementsByTagName("orientationPreferences");
+				element = (Element) nodes.item(0);
+				if(element!=null && element.getElementsByTagName("preference")!=null) {
+					for(int i=0; i<element.getElementsByTagName("preference").getLength(); i++){
+						Element e = ((Element)element.getElementsByTagName("preference").item(i));
+						
+						try {
+							if(!e.getAttribute("orientation").isEmpty()) {
+								orientationPreferencesMap.put(SexualOrientation.valueOf(e.getAttribute("orientation")), Integer.valueOf(e.getAttribute("value")));
+							}
+						} catch(IllegalArgumentException ex){
+						}
```

Comment:
shouldn't there be a warning here?

Reply:
There's lots of empty catch blocks in LT. Ideally, each catch block should report an error, or include a comment that describes why it's ok to ignore the error.